crates/nu-command/src/core_commands/do_.rs

```diff
@@ -95,87 +107,89 @@ impl Command for Do {
             block,
             input,
             call.redirect_stdout,
-            ignore_errors || capture_errors,
+            capture_errors,
```

Comment:
Is stderr really supposed be redirected when ignoring errors? I'll bring this back if so, but that seemed weird.

Reply:
Some tests fail without this so the answer is yes.